docs/endpoints/openrtb2/auction.md

```diff
@@ -290,6 +290,12 @@ This adds two optional properties:
 
 These fields will be forwarded to each Bidder, so they can decide how to process them.
 
+
+#### Viewability
+
+`bidrequest.imp[i].metric` is now supported. `type` and `score` are mandatory fields. The viewability vendors can be sent as a string array through `bidrequest.imp[i].ext.prebid.viewabilityvendors`
```

Comment:
Sorry I don't see the **score** field. Did you mean **value**?

Reply:
oops sorry! yes, it is value.

Reply:
Fixed it. Thanks! 👍